mm/kasan/CMakeLists.txt

```diff
@@ -19,7 +19,11 @@
 # ##############################################################################
 if(CONFIG_MM_KASAN)
   target_sources(mm PRIVATE kasan.c)
-  set_source_files_properties(kasan.c PROPERTIES COMPILE_FLAGS
-                                                 -fno-sanitize=kernel-address)
-  set_source_files_properties(kasan.c PROPERTIES COMPILE_FLAGS -fno-lto)
+
+  target_compile_options(mm PRIVATE -fno-sanitize=kernel-address)
+
+  if(NOT CONFIG_LTO_NONE)
```

Comment:
```suggestion
  if(CONFIG_LTO_NONE)
```

Reply:
@xiaoxiang781216 - look at the original Make.defs
https://github.com/apache/nuttx/blob/master/mm/kasan/Make.defs
Are you sure?

I think the goal is to disable LTO specifically for these files - if it was enabled globally in build.

Reply:
e.g. https://github.com/search?q=repo%3Aapache%2Fnuttx%20CONFIG_LTO_NONE&type=code

Reply:
yes, my mistake.